CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+# v1.11.0 (2022-11-15)
+
+## OS Changes
+
+* Prevent a panic `early-boot-config` when there is no IMDS Region ([#2493])
+* Update grub to 2.06-42 ([#2503])
+* Allow interface configuration via Ethernet permanent address ([#2519])
+* Allow bootstrap containers to manage swap ([#2537])
+* Add systemd analyze commands to troubleshooting log collection tool ([#2550])
+* Allow bootstrap containers to manage network configuration ([#2558])
+* Serialize bootconfig values correctly when the value is empty ([#2565])
+* Update zlib, libexpat, libdbus, docker-cli ([#2583])
+* Update host containers ([#2574])
+* Unmask /sys/firmware from host containers in host-ctr ([#2573])
+
+## Orchestrator Changes
+
+### ECS
+
+* Add additional ECS API configurations ([#2527])
+
+### Kubernetes
+
+* Update EKS-D to 1.23-6 for k8s-1.23 variants ([#2488])
+* Update EKS-D to 1.22-11 for k8s-1.22 variants ([#2490])
+* Add a timeout when calling EKS for configuration values ([#2566])
+* Enable IAM Roles Anywhere with the k8s `ecr-credential-provider` plugin ([#2377], [#2553])
+* Kubernetes updates ([#2578], [#2580], [#2582]),
```

Comment:
Let's combine all the Kubernetes updates for each version and indicate the "final" version that was shipped in the release note, even if we link to all the PRs. Right now the notes suggest that `1.23-6` and `1.22-11` are in the release, which is incorrect.